Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -40,20 +51,33 @@ extension Style {
 
     /// Filters through source to determine supported locale styles.
     /// This is needed for v7 support
-    internal func getLocaleValue(locale: Locale) -> String {
+    internal func getLocaleValue(locale: Locale) -> String? {
+        // Reference for supported languages: https://docs.mapbox.com/ios/navigation/guides/localization-and-internationalization/#supported-languages
+        let supportedLocaleIdentifiers = ["ar", "zh", "en", "fr", "de", "it", "ja", "ko", "pt", "ru", "zh-Hans", "es", "zh-Hant", "vi", "zh-Hant-TW"]
```

Comment:
Would love to see this array be sorted for readability.

```suggestion
        let supportedLocaleIdentifiers = ["ar", "de", "en", "es", "fr", "it", "ja", "ko", "pt", "ru", "vi", "zh", "zh-Hans", "zh-Hant", "zh-Hant-TW"]
```